Created print.md for Python: Built-in Functions (#476)
Codecademy/docs: content/python/concepts/built-in-functions/terms/print/print.md

---
Title: 'print()'

> Description: 'Outputs the string representation of an object.'

Reviewer: ```suggestion
Description: 'Print the string representation of an object.'
```

Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Functions'
  - 'Methods'
  - 'Strings'
CatalogContent:

>   - 'learn-python-3'
>   - 'paths/computer-science'
>   - 'paths/data-science'

Reviewer: ```suggestion
  - 'learn-python-3'
  - 'paths/computer-science'
```

Just one `learn-` slug and one `paths/` slug are sufficient.

---

> The `print()` function takes in a string parameter or any other element, such as an integer. The parameter is converted to a string through an implicit call of the built-in `str()` function and the value is outputted.

Reviewer: ```suggestion
The `print()` function accepts an object as a parameter, such as a string, a number, or a list. It is then converted to a string through an implicit call of the built-in `str()` function and the value is printed to an output stream.
```

## Syntax

```py
print(object)
```

## Example

```py

> print("Hello, World")

Reviewer: ```suggestion
print("Hello, World!")
# Output: Hello, World!
```

```

> 
> The output would be:
> 
> ```pseudo
> Hello, World!
> ```
> 

Reviewer: ```suggestion

```

> **Note:** To print an integer of a floating-point number, use the str() function.

Reviewer: Can we expand on this with, perhaps, a codebyte/py example?

Reviewer 2: @KabirCampwala @Dusch4593 I'm actually a bit confused by this note. When we print an integer or floating-point in Python, can't we just do (for example) `print(3)` or `print(3.14)` which would output 3 and 3.14, respectively?

Reviewer: True. Plus, we already have an entry for the `str()` function underway. Gonna remove this and include a codebyte example about using the `sep` and `end` parameters.

---
Title: 'print()'
Description: 'Takes in data such as a string or an int converted to string using str() and prints it to the standard output device.'
Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Functions'
  - 'Methods'
  - 'Strings'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
  - 'paths/data-science'
---

> The print() function takes in a string or any other element such as int converted to a string using str() and displays those values to the standard output device.

Reviewer 2: ```suggestion
The `print()` function takes in a string or any other element such as an integer converted to a string using `str()` and displays those values to the standard output device.
```

I get what this is trying to say 👍 however, it's not very clear at first glance. The `print()` function implicitly calls on `str()`. This is called type casting or type conversion and we have on Docs entry on that: [Python casting](https://www.codecademy.com/resources/docs/python/casting)! I suggest linking out to this source.

Let's re-write this sentence to explain that the `print()` function takes in an argument and displays it to the "standard output device (as you've stated)." Then, explain how the `print()` function works.

## Syntax

```py

> print("Hello, World!")

Reviewer 2: ```suggestion
print(object)
```

I recommend using a placeholder here. Take a look at the info under the "Syntax" heading for [w3schools](https://www.w3schools.com/python/ref_func_print.asp) or some other (built-in function)[https://www.codecademy.com/resources/docs/python/built-in-functions] Dos entries.

We can create another heading for the examples and include your line 22 as an ex. as well:

## Example 1

Short description:

```py
Code here
```

## Example 2

...and so on

```

The output would be:

> ```py

Reviewer 2: ```suggestion
```pseudo
```

Hello, World!
```

If one were to print an integer of a floating-point number they can do so by using the str() function:

```py

> print(str(3.14))

Reviewer 2: This works fine, though passing just 3.14 into the function displays the implicit call of `str()`. I'd recommend to not type cast the integer to a string but we can still explain what's going on like how you've done on line 31 for the current line 33-35 codeblock.

```

The output would be:

> ```py

Reviewer 2: ```suggestion
```pseudo
```
